duckduckgo/duckduckgo-privacy-extension
# ATB integration tests

**Reviewer:** @jdorweiler 

**CC:** @jkv 


## Description:

Adds integration tests to the ATB workflow (and more generally, an integration test harness).

## Steps to test this PR:
1. `npm run test-int`, tests should pass (`npm run test-int-x` if you've got a Linux box)
2. If you've got some time e.g. going off to lunch, you can run something like the following to run the tests repeatedly until one of them breaks:

```bash
$ while npm run test-int; do :; done
```

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
